spectator-ext-spark/src/main/resources/reference.conf

```diff
@@ -2,14 +2,13 @@
 spectator.spark {
   name-patterns = [
     // EXECUTORS
-    // 97278898-4bd4-49c2-9889-aa5f969a7816-0013.97278898-4bd4-49c2-9889-aa5f969a7816-S1/2.executor.filesystem.file.largeRead_ops
+    // 97278898-4bd4-49c2-9889-aa5f969a7816-0013.2.executor.filesystem.file.largeRead_ops
```

Comment:
To maintain compatibility, please add a block to support the new format and make the agent ID clause non-optional in the existing one.

For example (adjust as needed for valid `name` values in spark 2.+):
```
    // EXECUTORS
    // 97278898-4bd4-49c2-9889-aa5f969a7816-0013.97278898-4bd4-49c2-9889-aa5f969a7816-S1/2.executor.filesystem.file.largeRead_ops
    {
      pattern = "^([^.]+)\\.(([^.]+)\/)(\\d+)\\.((executor|jvm|CodeGenerator)\\..*)"
      name = 5
      tags = {
        "appId" = 1
        "agentId" = 3
        "executorId" = 4
      }
    },

    // 97278898-4bd4-49c2-9889-aa5f969a7816-0013.2.executor.filesystem.file.largeRead_ops
    {
      pattern = "^([^.]+)\\.(\\d+)\\.((executor|jvm|HiveExternalCatalog|CodeGenerator)\\..*)"
      name = 3
      tags = {
        "appId" = 1
        "executorId" = 2
      }
    },
```

Reply:
Done